dataflow/lib/exporters.py

```diff
@@ -0,0 +1,80 @@
+from functools import wraps
+
+def text(datasets, export_method="to_column_text", headers=None, concatenate=False):
+    import json
+    header_string = "#" + json.dumps(headers, separators=(',', ':'))[1:-1] + "\n"
+    outputs = []
+    if len(datasets) > 0:
+        exports = [getattr(d, export_method)() for d in datasets]
```

Comment:
The naming problem is ugly.  

For unpolarized reflectometry text files I want a plain name without '_unpolarized' coming from the entry name in the nexus file.  For polarized, I want a plain name with the four cross sections in the file, separated by blank lines.  Ideally the user would not have to download them with concatenate for them to appear in the same file. What should happen with concatenated polarized?

For hdf files that are not concatenated, just having a generic entry name would be better than dumping the file name into the entry name.  This naturally handles the polarization cross sections nicely.

For concatenated hdf the entry names must be unique, so name for each entry seems appropriate.  What about concatenated polarized---how do we bundle the associated polarization cross sections together?  Looks like you are only copying the first item of each concatenated file.  

Note that DCS is defining filename but it is ignored in this code.

Maybe keep things simple for now, and have the instrument code define only filename, not name, entry and suffix.

Reply:
I started with filename and the name entry and suffix pattern evolved from that because of the concatenation feature - the aggregator has to make the final name, so it needs the information for each of the entries and the suffix.  If you don't like the way the entry names get used in the concatenated text files we can certainly change that, but the pattern would be hard to change.

Reply:
I documented the following behaviour in core.DataType but didn't change the code to do so:

      use name+suffix if concatenated
      use name_entry+suffix if not concatenated

Please resolve by changing one or the other.